snappy/dpu_snappy.c

```diff
@@ -293,6 +312,11 @@ snappy_status snappy_uncompress_dpu(struct host_buffer_context *input, struct ho
 		DPU_ASSERT(dpu_copy_to(dpu, "output_offset", 0, output_offset[dpu_idx], sizeof(uint32_t) * NR_TASKLETS));
 		DPU_ASSERT(dpu_copy_to(dpu, "output_length", 0, &output_length, sizeof(uint32_t)));
 		DPU_ASSERT(dpu_copy_to(dpu, "output_buffer", 0, &output_buffer_start, sizeof(uint32_t)));
+		/* copy metrics just to basically zero the buffers out? */
```

Comment:
This comment doesn't sound very confident. Is that why you are copying the buffers?